test/models/post_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class PostTest < ActiveSupport::TestCase
```

Comment:
Please write the appropriate test for the post (such as the validations and association with the user model)